src/ui/units/ql/containers/QL/QLActionPanel/useQLActionPanel.ts

```diff
@@ -75,37 +73,34 @@ export const useQLActionPanel = (args: UseQlActionPanelArgs): AdditionalButtonTe
             },
         ];
 
-        if (enableEditHistoryQL) {
-            items = [
-                {
-                    key: 'undo',
-                    action: handleClickGoBack,
-                    className: b('undo-btn'),
-                    icon: {data: ArrowUturnCcwLeft, size: 16},
-                    view: 'flat',
-                    disabled: !canGoBack,
-                    qa: QLPageQA.UndoButton,
-                    title: i18n('component.action-panel.view', 'button_undo'),
-                    hotkey: UNDO_HOTKEY.join('+'),
-                },
-                {
-                    key: 'redo',
-                    action: handleClickGoForward,
-                    className: b('redo-btn'),
-                    icon: {data: ArrowUturnCwRight, size: 16},
-                    view: 'flat',
-                    disabled: !canGoForward,
-                    qa: QLPageQA.RedoButton,
-                    title: i18n('component.action-panel.view', 'button_redo'),
-                    hotkey: REDO_HOTKEY.join('+'),
-                },
-                ...items,
-            ];
-        }
+        items = [
+            {
+                key: 'undo',
+                action: handleClickGoBack,
+                className: b('undo-btn'),
+                icon: {data: ArrowUturnCcwLeft, size: 16},
+                view: 'flat',
+                disabled: !canGoBack,
+                qa: QLPageQA.UndoButton,
+                title: i18n('component.action-panel.view', 'button_undo'),
+                hotkey: UNDO_HOTKEY.join('+'),
+            },
+            {
+                key: 'redo',
+                action: handleClickGoForward,
+                className: b('redo-btn'),
+                icon: {data: ArrowUturnCwRight, size: 16},
+                view: 'flat',
+                disabled: !canGoForward,
+                qa: QLPageQA.RedoButton,
+                title: i18n('component.action-panel.view', 'button_redo'),
+                hotkey: REDO_HOTKEY.join('+'),
+            },
+            ...items,
```

Comment:
There is no need to spread the array - you can define it immediately as needed

Reply:
Ok, I'll fix this in next PR, when I will remove this flag completely